pkg/cmd/repo_list.go

```diff
@@ -39,16 +38,19 @@ func newRepoListCmd(out io.Writer) *cobra.Command {
 		ValidArgsFunction: noMoreArgsCompFunc,
 		RunE: func(_ *cobra.Command, _ []string) error {
 			f, _ := repo.LoadFile(settings.RepositoryConfig)
-			if len(f.Repositories) == 0 && !(outfmt == output.JSON || outfmt == output.YAML) {
-				return errors.New("no repositories to show")
+			if len(f.Repositories) == 0 {
+				switch outfmt {
+				case output.Table:
+					_, _ = fmt.Fprintln(out, "no repositories to show")
+					return nil
+				case output.JSON, output.YAML:
+					return outfmt.Write(out, &repoListWriter{repos: []*repo.Entry{}})
+				}
 			}
-
 			return outfmt.Write(out, &repoListWriter{f.Repositories})
 		},
 	}
-
```

Comment:
Can we revert these unrelated (and unintended?) whitespace changes please.

(it is bad practice to include multiple unrelated changes into a single PR: it often makes the PR harder to review/reviewer needs to juggle multiple concerns, and commingles the intent of the different changes)